script.js

```diff
@@ -160,4 +160,19 @@ document.addEventListener('DOMContentLoaded', () => {
   }
 
   generateProjects();
-});
\ No newline at end of file
+});
+
+//  CONTACT ME FORM VALIDATION SCRIPT
+// eslint-disable-next-line no-unused-vars
+const formValidation = () => {
```

Comment:
- [ ] As required by Microverse, you are not meant to use any linters disable rule on your code unless otherwise stated, I'll suggest you remove the linters disable and try to fix the issue. Following the suggestion I gave above will also resolve this issue. :smile: :+1: